assisted-events-scrape/utils/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .logger import log
+from .counters import ErrorCounter, Changes
```

Comment:
Need to declare `__all__` for defining what symbols in a module will be exported